Networking/Networking/Model/ShippingLabel/Packages/Carriers and Rates/ShippingLabelPackageSelected.swift

```diff
@@ -21,6 +22,7 @@ public struct ShippingLabelPackageSelected: Equatable, GeneratedFakeable {
                 height: Double,
                 weight: Double,
                 isLetter: Bool,
+                hazmatCategory: String,
```

Comment:
Good suggestion, @jaclync! I have just one question about it: Since the `ShippingLabelPackageSelected` is contained inside the `Networking` module, am I allowed to import a type from the `WooCommerce` module inside of it? I was wondering if this would require a Network layer version of the `ShippingLabelHazmatCategory`. WDYT?